src/add-hooks.coffee

```diff
@@ -13,6 +14,8 @@ mergeSandboxedHooks = require './merge-sandboxed-hooks'
 addHooks = (runner, transactions, callback) ->
   # Note: runner.configuration.options must be defined
 
+  customConfigCwd = runner?.configuration?.custom?.cwd
```

Comment:
comment above said at least `runner.configuration` is always accessible, so no need to `runner?.configuration?`